Ansible documentation 0.x

dLobatog opened the pull request:
Aside from the manual, it also changes some parts of the front page and
the manual in order to acknowledge Ansible reporting and facts.

dLobatog commented:
Pending https://github.com/theforeman/foreman-packaging/pull/1012

mmoll commented on plugins/foreman_ansible/0.2/index.md:
```diff
@@ -0,0 +1,96 @@
+
+* And run this line to download the callback file in the appropriate location
+
+      wget https://raw.githubusercontent.com/theforeman/foreman_ansible/master/extras/foreman_callback.py -O ~/.ansible/plugins/callback_plugins/foreman_callback.py
```
maybe use the version here?

mmoll commented on plugins/foreman_ansible/0.2/index.md:
```diff
@@ -0,0 +1,96 @@
+* Now open the script to set up `FOREMAN_URL` and `FOREMAN_SSL_CERT` variables on the callback. Alternatively, you can find the source for the [Ansible callback here](https://github.com/theforeman/foreman_ansible) and deploy it in your directory of choice.
+
+## 2.2 Plugin
+### 2.2.1 Installer (recommended on Foreman 1.11+)
```
I'd rather wait for this, unitl it's clear (and merged into -installer) what's needed on the core and and the proxy side. At least it should be clear that there is a distinction between core and proxy.

dLobatog replied:
On the proxy side - nothing - there's no proxy component for the moment, at least as far as foreman_ansible is concerned. Remote execution is a whole another thing that is not merged yet. But I'll remove the installer section until 1.11 then!

mmoll commented on plugins/foreman_ansible/0.2/index.md:
```diff
@@ -0,0 +1,96 @@
+
+      wget https://raw.githubusercontent.com/theforeman/foreman_ansible/master/extras/foreman_callback.py -O ~/.ansible/plugins/callback_plugins/foreman_callback.py
+
+* Now open the script to set up `FOREMAN_URL` and `FOREMAN_SSL_CERT` variables on the callback. Alternatively, you can find the source for the [Ansible callback here](https://github.com/theforeman/foreman_ansible) and deploy it in your directory of choice.
```
a link or more information regarding SSL certs would be nice. I suppose a trusted cert is needed?

dLobatog replied:
Yup, not sure what else can I write..?

dLobatog commented:
I updated this except for the SSL certs part - not sure what else to comment there

domcleal commented on plugins/foreman_remote_execution/0.2/index.md:
```diff
@@ -31,7 +31,7 @@ A few examples of tasks that this plugin helps you achieve:
 - Install/update/remove a package
 - Bootstrap a configuration management agent
-- Trigger a Puppet/Salt/Chef run
+- Trigger a Puppet/Salt/Chef/Ansible run
```
Note that the foreman_remote_execution plugin maintains a nightly/ version that's used to cut their next release, you should probably update that (either instead, or as well as, if they want this change).

dLobatog replied:
Good call actually, it's not even merged yet

dLobatog committed:
Ansible documentation 0.x

Aside from the manual, it also changes some parts of the front page and
the manual in order to acknowledge Ansible reporting and facts.

dLobatog commented:
Added a bit about configuring SSL certs and URL via environment variables thanks to @agx 's latest addition to the plugin

mmoll commented on plugins/foreman_ansible/0.x/index.md:
```diff
@@ -0,0 +1,101 @@
+
+* And run this line to download the callback file in the appropriate location
+
+      wget https://raw.githubusercontent.com/theforeman/foreman_ansible/0.2.2/extras/foreman_callback.py -O ~/.ansible/plugins/callback_plugins/foreman_callback.py
```
0.3?

dLobatog commented:
@mmoll Updated to point the callback file to 0.3, thanks !

dLobatog commented:
@mmoll Actually ended up changing it to master instead of 0.3. There are a few key changes in master that fix bugs with Ansible Tower & default SSL setup - https://github.com/theforeman/foreman_ansible/commits/master/extras/foreman_callback.py

mmoll commented:
merged, gracias @dLobatog!
